frontend/src/scenes/dashboard/dashboardLogic.ts

```diff
@@ -342,15 +343,14 @@ export const dashboardLogic = kea<dashboardLogicType<DashboardLogicProps>>({
                 if (!items || !items.length) {
                     return null
                 }
-                let lastRefreshed = items[0].last_refresh
-
+                let oldestLastRefreshed = items[0].last_refresh ? dayjs(items[0].last_refresh) : null
                 for (const item of items) {
-                    if (item.last_refresh && (!lastRefreshed || item.last_refresh < lastRefreshed)) {
-                        lastRefreshed = item.last_refresh
+                    const itemLastRefreshed = item.last_refresh ? dayjs(item.last_refresh) : null
+                    if (!oldestLastRefreshed || (itemLastRefreshed && itemLastRefreshed < oldestLastRefreshed)) {
```

Comment:
Can you use `<`? The dayjs docs seem to mention `.isBefore()`, but I didn't look too deep. I'd assume `.valueOf()` turns it into a number comparison?

Reply:
From playing with the library, I think both work, but I just moved to using `.isBefore` because why would they include it if they didn't want people using it....🤔